examples/src/main/scala/org/apache/spark/examples/ml/LogisticRegressionSummaryExample.scala

```diff
@@ -30,11 +30,11 @@ object LogisticRegressionSummaryExample {
   def main(args: Array[String]): Unit = {
     val conf = new SparkConf().setAppName("LogisticRegressionSummaryExample")
     val sc = new SparkContext(conf)
-    val sqlCtx = new SQLContext(sc)
-    import sqlCtx.implicits._
+    val sqlContext = new SQLContext(sc)
+    import sqlContext.implicits._
```

Comment:
@jkbradley     
69: val bestThreshold = fMeasure.where($"F-Measure" === maxFMeasure)
70:      .select("threshold").head().getDouble(0)

Still needs the import. Otherwise, it won't be compiled.